README.md

```diff
@@ -1,4 +1,96 @@
-MolSim
-===
+# Molecular Dynamics - Group A
 
-The Molecular Dynamics teaching code.
+This repository is for the practical course **PSE: Molecular Dynamics** for group A in the SS 2024 semester.
+
+## All Contributors
+
+**Group Members**
+
+- [Ashutosh Solanki](https://github.com/AshIsAtWork)
+- [Daniel Schade](https://github.com/D4ni3lSch4d3)
+- [Robin Cleve](https://github.com/cleveee)
+
+**Course Supervisors**
+
+- [Manish Mishra](https://github.com/manishmishra6016)
+- [Markus Mühlhäußer](https://github.com/thesamriel)
+- [Fabio Gratl](https://github.com/FG-TUM)
+
+## About The Project
+
+This is the practical project repository for the course offered by the Scientific Computing Chair at TUM.
+
+**Built Using:**
+
+- Clang 16.0.6
+- CMake 3.27.4
+- make 4.3
+- Boost Program Options 1.74.0.3
+
+**Dependencies**
+
+ou will need the following to compile the program:
+
+- Doxygen 1.10.0
+
+```bash
+sudo apt install doxygen
+```
+
+- Graphviz 2.42.2
+
+```bash
+sudo apt install graphviz
+```
```

Comment:
Are those actually needed to compile the (main) program? I would add an additional point, that these are needed if one would like to build the doxygen documentation.
That is also exactly the reason why we want an option to disable documentation in CMake.